infra/test/setup/main.tf

```diff
@@ -30,9 +30,3 @@ module "project" {
     "serviceusage.googleapis.com"
   ]
 }
-
-resource "google_compute_network" "vpc_network" {
-  project                 = module.project.project_id
-  name                    = "default"
-  auto_create_subnetworks = true
-}
```

Comment:
This `infra/test/setup/main.tf` creates the project used by integration tests.
Currently, we create a VPC network called `default` inside that project.
But since we will be using a new, custom VPC in the JSS Terraform, we no longer need this `default` VPC network.